azimuth/plots/dataset_warnings.py

```diff
@@ -167,6 +165,123 @@ def min_nb_samples_plot(
+        alert_eval: Indicator if there is an alert per label in the evaluation set.
+        cls_names: Class names.
+
+    Returns:
+        Plot.
+
+    """
+    order = eval_nb_sample.argsort()
+
+    fig = nb_samples_plot(
+        train_nb_sample, eval_nb_sample, alert_train, alert_eval, cls_names, order
+    )
+    fig.add_vline(x=threshold, line_width=1, line_dash="dot", line_color=Colors.Orange)
+    fig.add_annotation(
+        text=threshold,
+        x=threshold,
+        y=1.002,
+        yanchor="bottom",
+        yref="paper",
+        font=dict(color=Colors.Orange),
+        showarrow=False,
+    )
+
+    return DatasetWarningPlots(overall=json.loads(fig.to_json()), per_class=None)
+
+
+def class_imbalance_plot(
+    train_nb_sample: np.ndarray,
+    eval_nb_sample: np.ndarray,
+    train_mean: float,
+    eval_mean: float,
+    max_perc_delta: float,
+    alert_train: List[bool],
+    alert_eval: List[bool],
+    cls_names: List[str],
+) -> DatasetWarningPlots:
+    """Generate the plot for the class imbalance warnings.
+
+    Args:
+        train_nb_sample: Distribution of the number of samples per label in the training set.
+        eval_nb_sample:  Distribution of the number of samples per label in the eval set.
+        train_mean: Mean of the number of samples in the training set.
+        eval_mean: Mean of the number of samples in the evaluation set.
+        max_perc_delta: Percentage around the mean where values need to be.
+        alert_train: Indicator if there is an alert per label in the training set.
+        alert_eval: Indicator if there is an alert per label in the evaluation set.
+        cls_names: Class names.
+
+    Returns:
+        Plot.
+
+    """
+    order = eval_nb_sample.argsort()
+
+    fig = nb_samples_plot(
+        train_nb_sample, eval_nb_sample, alert_train, alert_eval, cls_names, order
+    )
+    num_classes = len(cls_names)
+    common_args = dict(layer="below", line_width=1, y0=-1)
+    common_args_line = dict(type="line", line_dash="dot", **common_args)
+    fig.add_shape(
+        x0=eval_mean,
+        x1=eval_mean,
+        y1=num_classes + 2,
+        line_color=Colors.DataViz1,
+        **common_args_line,
+    )
+    fig.add_shape(
+        x0=train_mean,
+        x1=train_mean,
+        y1=num_classes + 1,
+        line_color=Colors.DataViz2,
+        **common_args_line,
+    )
+    common_args_rect = dict(type="rect", opacity=0.3, **common_args)
+    fig.add_shape(
+        x0=eval_mean - max_perc_delta * eval_mean,
+        x1=eval_mean + max_perc_delta * eval_mean,
+        y1=num_classes + 2,
+        fillcolor=Colors.DataViz1,
+        line_color=Colors.DataViz1,
+        **common_args_rect,
+    )
+    fig.add_shape(
+        x0=train_mean - max_perc_delta * train_mean,
+        x1=train_mean + max_perc_delta * train_mean,
+        y1=num_classes + 1,
+        fillcolor=Colors.DataViz2,
+        line_color=Colors.DataViz1,
```

Comment:
Do those `line_color`s have any effect? If so, is this a typo?
```suggestion
        line_color=Colors.DataViz2,
```

Reply:
Good catch, typo! I thought they did a border around the rectangle, but I'm not sure anymore. I'll double-check.

Reply:
They didn't do anything :)

Reply:
Oh no, they did!